Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -193,6 +193,40 @@ Command keywords and values
          make a copy of the record on the same page.
 
 
+ - :Command:
+         copyToLanguage
+   :Type:
+         integer
+   :Value:
+         This DataHandler command is similar to the 'localize' command: Both, record and child records,
+         are copied to the given language. But in contrast to 'localize the 'copyToLanguage' command 
+         does not set transOrigPointerField fields like l10n_parent.
```

Comment:
I'd still love to see PHP highlighting for `transOrigPointerField` and SQL highlighting for `l10n_parent`.